### ISSUE TITLE
How to train with 2 GPUs?

### ISSUE THREAD

**Issue by Liam-Xie** [yoxu515/aot-benchmark#13]
Thanks for your excellent work!

I want to re-train AOT, however, I only have two 3090 that cannot support the large batch size in the paper.  In that case, I need to half the batch size or even reduce it to 6 since the memory of 3090 is not as large as V100. May I have some suggestions on the hyperparameters if I reduce the batch size, e.g., should I also half the training lr and double the iterations as well? 

BTW, do you think reducing batch size will reduce the performance of  AOT?

**Comment by z-x-yang** [comment]
Not sure what will happen if you use a batch size smaller than the default.

To keep the same batch size with less memory usage, a desirable manner is to use gradient accumulation.

**Comment by Liam-Xie** [comment]
Thanks. It is a good idea.
